main.py

```diff
@@ -1,7 +1,50 @@
 import init_django_orm  # noqa: F401
 
-from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
-def main() -> QuerySet:
-    pass
+def main() -> None:
+    list_genre = ["Western", "Action", "Dramma"]
+    dict_actors = [
+        {"first_name": "George", "last_name": "Clooney"},
+        {"first_name": "Keanu", "last_name": "Reeves"},
+        {"first_name": "Scarlett", "last_name": "Keegan"},
+        {"first_name": "Will", "last_name": "Smith"},
+        {"first_name": "Jaden", "last_name": "Smith"},
+        {"first_name": "Scarlett", "last_name": "Johansson"}
+    ]
+
+    # Create GENRE with list genre
+
+    for genre in list_genre:
+        Genre.objects.create(genre=genre)
```

Comment:
The `Genre.objects.create()` method is using `genre=genre`, which is incorrect. It should use `name=genre` to match the field name specified in the `Genre` model.